.github/workflows/release-standalone-package.yml

```diff
@@ -13,7 +13,7 @@ on:
           - '@n8n/scan-community-package'
 
 env:
-  BUILD_CACHE_KEY: ${{ github.sha }}-${{ github.event.inputs.package }}-build
+  BUILD_DIST_NAME: build-dist-${{ github.sha }}
```

Comment:
Doesn't it still need package in the cache key? because I'm only building the specific package